dbrp/service_auth.go

```diff
@@ -0,0 +1,51 @@
+package dbrp
+
+import (
+	"context"
+
+	"github.com/influxdata/influxdb/v2"
+	"github.com/influxdata/influxdb/v2/authorizer"
+)
+
+var _ influxdb.DBRPMappingServiceV2 = (*DBRPMappingAuthorzedService)(nil)
+
+type DBRPMappingAuthorzedService struct {
```

Comment:
Missing an `i` should be `DBRPMappingAuthorizedService`

Reply:
boom George the eagle